thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -1197,6 +1197,7 @@ class CDiskGroupAggregateSlave
 
                 if (!container.queryLocalOrGrouped() && container.queryJob().querySlaves()>1)
                 {
+                    BlockedActivityTimer t(totalCycles, timeActivities);
```

Comment:
I think this should now be removed, there isn't really any 'blocked on others' time here. It is all local setup.
(NB: even the distributor->connect, isn't actually connecting..)